api/valor_api/schemas/filters.py

```diff
@@ -303,3 +305,28 @@ class Filter(BaseModel):
         extra="forbid",
         protected_namespaces=("protected_",),
     )
+
+
+def field_to_tuple(field_info):
```

Comment:
is this function used anywhere? the name doesn't match the output to me; I'm wondering if you forgot to delete these lines

Reply:
good catch-- nope deleted